src/shared/actions/flags.ts

```diff
@@ -62,3 +73,19 @@ export const getFlags = () => async (
     dispatch(setFlags(RemoteDataState.Error, null))
   }
 }
+
+export const getPublicFlags = () => async (dispatch: Dispatch<Actions>) => {
```

Comment:
this isn't an action creator, since it has side effects. it (and `getFlags`) should be moved to a thunks file. we had problems in the pace with race conditions on load because of circular dependencies. what happens is the reducer includes the action creator. normally this isn't a problem as most action creators don't have a lot of dependencies on outside things. thunks do, however, and these dependencies getting loaded is what causes the problems, because the dependency graph kind of explodes out.

i think dealing with these circular dependencies will make race conditions less likely and will make the system behave more consistently locally and out on the web.